src/symbolize/libbacktrace.rs

```diff
@@ -381,7 +395,7 @@ unsafe fn init_state() -> *mut bt::backtrace_state {
             unsafe fn query_full_name(buf: &mut [i8]) -> Result<&[i8], ()> {
                 let p1 = OpenProcess(PROCESS_QUERY_INFORMATION, FALSE, GetCurrentProcessId());
                 let mut len = buf.len() as u32;
-                let rc = QueryFullProcessImageNameA(p1, 0, buf.as_mut_ptr(), &mut len);
+                let rc = GetModuleFileNameExA(p1, ptr::null_mut(), buf.as_mut_ptr(), &mut len);
```

Comment:
To confirm, this function has similar portability and version compatibility as the previous?

Reply:
`QueryFullProcessImageNameA` is available starting from Windows Vista and isn't available on UWP
`GetModuleFileNameExA` is available starting from Windows XP, and is available on UWP
So switching would definitely not impact any targeted users.